platform/android/export/export.cpp

```diff
@@ -597,7 +597,7 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			if (abi_index != -1) {
 				exported = true;
 				String abi = abis[abi_index];
-				String dst_path = "lib/" + abi + "/" + p_so.path.get_file();
+				String dst_path = "lib/" + abi.plus_file(p_so.path.get_file());
```

Comment:
Again not sure if this should be `"lib".plus_file(abi).plus_file(p_so.path.get_file())`